docs/security.md

```diff
@@ -0,0 +1,48 @@
+
+Exclusion should also be a concern at the forefront of your mind.  Oftentimes, you will be spending multiple coins in the same bundle, and they should all be tied together into one aggregated signature.  If you have good reason not to sign one of them, make sure you know what happens if it gets excluded from the bundle.  Furthermore, aggregated signatures can't be disaggregated into smaller signatures, but if you have previously signed one of the smaller combinations of public key-message pairs in the bundle, you can exclude the rest and reuse the smaller signature again on the remaining transactions.  This is known as **signature subtraction** and is another great reason to use `AGG_SIG_ME` as much as possible.
+
+## Puzzle and solution reveals
+
+Remember to think about when puzzles and solutions are revealed.  They are revealed only at spend time of the coin that is committed to them.  The only thing that the network sees prior to that is the parent coin and the puzzle hash.  This can be an advantage, since you can hide sensitive information for spending the coin inside the puzzle hash before it is ever revealed.  However, once the puzzle is revealed, it's revealed forever, so that sensitive information cannot be considered sensitive again.
+
+Also keep in mind that if a parent coin is currying information to its child coin before it creates it, that will be public before the child coin is spent.  For some wallets, this is an advantage since you may want certain data about a coin's puzzle to calculate whether or not it's yours.  However, if you were trying to use a plain-text password, that won't be very secure.  Instead, make sure to pre commit to things with hashes and then assert that they are reveled correctly later.
+
+## Password Locked Coin Security
+
+It's worth noting that the [password locked coin we've been building](https://chialisp.com/docs/common_functions#outer-and-inner-puzzles) is actually not very secure.  When you solve the puzzle, you have to reveal the password.  Since any full nodes whom you give your spend to will now be able to see your password, they can change the solution and pay themselves all the money instead!
+
+In order to fix it, it's probably best to curry in a public key that also has to sign for the solution.  The new puzzle will be able to be spent only with a password AND only by the person who you have decided owns this coin. Of course, this is not particularly useful most of the time and is usually about as good as a signature locked coin with extra steps.  Signatures are by far the most secure way to lock up your coins.
```

Comment:
You can wrap `and` in "*" to make it bold if you prefer that to uppercasing. (I think that's what uppercasing is supposed to simulate.)